config/jobs/kubernetes/publishing-bot/publishing-bot-presubmits.yaml

```diff
@@ -47,6 +47,30 @@ presubmits:
     annotations:
       testgrid-dashboards: sig-release-publishing-bot
       testgrid-tab-name: validate-rules
+  - name: pull-publishing-bot-test-kubernetes-master
+    always_run: false
+    decorate: true
+    annotations:
+      testgrid-dashboards: sig-release-publishing-bot
+      testgrid-tab-name: pull-publishing-bot-test-k8s-master
+      description: test publishing bot against k/k master
+    labels:
+      preset-dind-enabled: "true"
+    path_alias: k8s.io/publishing-bot
+    spec:
+      containers:
```

Comment:
We should set up cpu/memory requests and limits as per:

https://github.com/kubernetes/publishing-bot/blob/bd68f8104bf4615ae609226e8a9579127d035941/configs/kubernetes#L5-L8

Reply:
Done.